news/2017-07-10-osucatch-world-cup-2017-results.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "osu!catch World Cup 2017 - Results"
+permalink: http://osunews.tumblr.com/post/162821221298/osucatch-world-cup-2017-results
+date: 2017-07-10 12:30:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/162821221298/osucatch-world-cup-2017-results
+---
+
```

Comment:
needs "header" paragraph before the actual header image, usually a brief 2-3 line snippet of what the article contains at a glance